src/css/imports/mixins.less

```diff
@@ -58,6 +58,22 @@
     }
 }
 
+.safari-only-captions-position(@margin, @translation) {
+
+    // this detects Safari 9+
+    // http://browserstrangeness.bitbucket.org/css_hacks.html#webkit
+
+    @supports ((overflow:-webkit-marquee) and (justify-content:inherit)) or (hanging-punctuation:allow-end) {
+        video::-webkit-media-text-track-container {
+            top: @margin;
```

Comment:
What is margin? Is there any harm in adding `top: 0` captions.less? It would be better to use rule that work everywhere.